src/Changelog/BumpAndCommitChangelogVersionViaKeepAChangelog.php

```diff
@@ -66,11 +66,14 @@ public function __invoke(
         Assert::stringNotEmpty($newVersion);
         $bumper->updateChangelog($newVersion);
 
+        $message = sprintf(self::COMMIT_TEMPLATE, $newVersion, self::CHANGELOG_FILE, $newVersion);
+        Assert::stringNotEmpty($message);
```

Comment:
I think this is just `Assert::notEmpty()`, since `sprintf(): string` already enforces it to be a `string` 

Doesn't psalm complain about this one?

Reply:
It didn't - it complained that it couldn't determine it was non-empty, which is why I added the assertion. I'll update to simply `notEmpty()` and see what happens.